### PR TITLE
add new dresden usergroup

### PR DESCRIPTION
**Pull request by sarah-ki**
Hello, I am helping to organize a brand new Dresden Ruby usergroup after hearing this year's inspiring Lightning talk at Euruko in Sarajevo. May we be added to your repo, please?

### CODE REVIEW

File: config/locales/es.label.yml [hunk 1]
```diff
@@ -115,3 +115,9 @@ es:
       subtitle: 'antes: Barcelona on Rails'
       meta_desc: Grupo de usuarios de Ruby en Barcelona
       custom_recurrence: Tercer jueves de cada mes a las 19:00
+    dresdenrb:
+      :city: City
```
Comment (salzig) [inline config/locales/es.label.yml:119]: Please provide correct values for Spanish and Polish too :)

**Comment by phoet** [comment]
Is the lightningtalk available somewhere?

**Comment by JoschkaSchulz** [comment]
Hey there cool to hear from you from Dresden. I talked with Richard (we both also attended the Ruby Kaigi this year) and he told me that you plan to create a usergroup. Really happy to see that you setting everything up :)

@phoet everything got recorded as I remember right but it seems that it isn't uploaded until now.

**Comment by salzig** [comment]
@phoet thanks, got that mail too, but I don't have access to DNS.

**Comment by phoet** [comment]
Do have or want to setup a new dns magement account ie cloudflare, so that the domain onruby.de and onruby.eu can be managed on your side.

File: config/locales/es.label.yml (hunk 1 above)
Comment (sarah-ki) [reply config/locales/es.label.yml]: Done, thank you.

**Comment by salzig** [comment]
@phoet I already have an existing account with the email address you already know. Not sure if @JoschkaSchulz and/or @janz93 have one.

**Comment by phoet** [comment]
@salzig which nameserver address?

**Comment by salzig** [comment]
@phoet you mean the name target address?

**Comment by salzig** [comment]
@phoet I'm a bit confused, so: I expected that you want to add us as members for the domain. But maybe your intention is to move the domains out of your control?

**Review by JoschkaSchulz — APPROVED** [review]

**Comment by salzig** [comment]
Welcome https://dresdenrb.onruby.de

**Comment by sarah-ki** [comment]
Thank you very much, @salzig. Could you add admin privileges aswell to the sarah-ki Account and rug-dd@mailbox.org so we can start editing posts on https://dresdenrb.onruby.de , please?

**Comment by salzig** [comment]
@sarah-ki if you want to show up as orga on your UG page, the nicknames have to match what you wrote into whitelabel.yaml

**Pull request merged**